LBHFSSPublicAPI/serverless.yml

```diff
@@ -2,6 +2,7 @@ service: fss-public-api
 provider:
   name: aws
   runtime: dotnetcore3.1
+  timeout: 15
```

Comment:
I think we should document why this was changed in the Pull Request Notes section as the problem causing this might reappear in the future.

Reply:
Ah I missed this one, yes should be added.

Reply:
added a note in the PR about this.